cli/wallet/nep17.go

```diff
@@ -413,7 +413,12 @@ func transferNEP17(ctx *cli.Context) error {
 	defer wall.Close()
 
 	fromFlag := ctx.Generic("from").(*flags.Address)
-	from := fromFlag.Uint160()
+	var from util.Uint160
+	if fromFlag.IsSet {
+		from = fromFlag.Uint160()
+	} else {
+		from = wall.GetChangeAddress()
+	}
```

Comment:
We have the same problem with `multiTransferNEP17`, so it can also be fixed.

Reply:
Fixed.